[2019-11-17 09:14:45,602] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_dag.Load_projects_dim_table 2018-11-03T16:00:00+00:00 [queued]>
[2019-11-17 09:14:45,607] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_dag.Load_projects_dim_table 2018-11-03T16:00:00+00:00 [queued]>
[2019-11-17 09:14:45,607] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 09:14:45,607] {taskinstance.py:839} INFO - Starting attempt 4 of 4
[2019-11-17 09:14:45,607] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 09:14:45,612] {taskinstance.py:859} INFO - Executing <Task(LoadDimensionOperator): Load_projects_dim_table> on 2018-11-03T16:00:00+00:00
[2019-11-17 09:14:45,612] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'capstone_dag', 'Load_projects_dim_table', '2018-11-03T16:00:00+00:00', '--job_id', '208', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/capstone_dag.py', '--cfg_path', '/var/folders/8q/nyxhvd6d55nfqjdbrybk6jpw0000gn/T/tmpc5_a7ti1']
[2019-11-17 09:14:46,329] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table /Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2019-11-17 09:14:46,329] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   """)
[2019-11-17 09:14:46,415] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table [2019-11-17 09:14:46,415] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-11-17 09:14:46,725] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table [2019-11-17 09:14:46,725] {dagbag.py:90} INFO - Filling up the DagBag from /Users/likejun/airflow/dags/capstone_dag.py
[2019-11-17 09:14:46,738] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table [2019-11-17 09:14:46,738] {cli.py:516} INFO - Running <TaskInstance: capstone_dag.Load_projects_dim_table 2018-11-03T16:00:00+00:00 [running]> on host likejundeMacBook-Pro.local
[2019-11-17 09:14:46,745] {load_dimensions.py:23} INFO - LoadDimensionOperator for projects
[2019-11-17 09:14:46,749] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:14:46,749[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 09:14:48,782] {load_dimensions.py:34} INFO - get 0 records from projects
[2019-11-17 09:14:48,785] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:14:48,785[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 09:14:52,575] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:14:52,574[0m] {[34mdbapi_hook.py:[0m171} INFO[0m - DELETE FROM projects[0m
[2019-11-17 09:14:53,014] {taskinstance.py:1051} ERROR - relation "projects" does not exist
Traceback (most recent call last):
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
    pg_hook.run(f"DELETE FROM {self.table}")
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
    cur.execute(s)
psycopg2.ProgrammingError: relation "projects" does not exist

[2019-11-17 09:14:53,015] {taskinstance.py:1080} INFO - All retries failed; marking task as FAILED
[2019-11-17 09:14:53,035] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table Traceback (most recent call last):
[2019-11-17 09:14:53,035] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/bin/airflow", line 32, in <module>
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     args.func(args)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     return f(*args, **kwargs)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     _run(args, dag, ti)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     pool=args.pool,
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     return func(*args, **kwargs)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     result = task_copy.execute(context=context)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     pg_hook.run(f"DELETE FROM {self.table}")
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table     cur.execute(s)
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table psycopg2.ProgrammingError: relation "projects" does not exist
[2019-11-17 09:14:53,036] {base_task_runner.py:115} INFO - Job 208: Subtask Load_projects_dim_table 
[2019-11-17 09:14:55,606] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:14:55,605[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
